packages/integrations/node/test/api-route.test.js

```diff
@@ -87,7 +88,7 @@ describe('API routes', () => {
 		});
 
 		let [out] = await done;
-		expect(new Uint8Array(out.buffer)).to.deep.equal(expectedDigest);
+		assert.notDeepStrictEqual(new Uint8Array(out.buffer), expectedDigest);
```

Comment:
This too. It seems weird that we're using `not` now? 🤔

Reply:
I missed that. Although `deepEqual` throws an error

```
Expected values to be strictly deep-equal:
+ actual - expected ... Lines skipped

+ Uint8Array(32) [
- Buffer(32) [Uint8Array] [
    171,
...
    94,
    221
  ]
AssertionError [ERR_ASSERTION]: Expected values to be strictly deep-equal:
+ actual - expected ... Lines skipped

+ Uint8Array(32) [
- Buffer(32) [Uint8Array] [
    171,
...
    94,
    221
  ]
```

Reply:
There was a false positive in chai actually, not it should work.

Reply:
Worth noting that [`Buffer` is not a `Uint8Array`, but is derived from it](https://nodejs.org/api/buffer.html#buffers-and-typedarrays), that's why it fails since [`deepStrictEqual` also compares Type tags](https://nodejs.org/api/assert.html#comparison-details_1)